utils/content-utils.ts

```diff
@@ -114,8 +114,8 @@ function findPrevTopic(
 
 const topicTree = buildTopicTree(allTopics)
 export function getTopicData(slug: string): {
-    prev: { path: string; title: string } | undefined
-    next: { path: string; title: string } | undefined
+    prev: { path: string; title: string }
```

Comment:
```suggestion
    prev?: { path: string; title: string }
    next?: { path: string; title: string }
```
deployment is failing as `prev` and `next` aren't set as optional `?`

Reply:
ok, added your commit, you can re-deploy now